jeta/logger.py

```diff
@@ -0,0 +1,153 @@
+import logging
+import os
+import time
+from datetime import timedelta
+
+import pandas as pd
+import tensorflow as tf
+import wandb
```

Comment:
can we do the lazy import for TensorFlow and wandb? Since these libraries will only be used in Iogging, I think having them as optional dependencies makes sense.

Reply:
It's a good idea. We can actually use lazy importing for most of the common dependencies here at a later point. But I don't mind doing this for wandb and tf now

Reply:
+ pandas. 


Other modules are present in python 3 by default i suppose.